setup.py

```diff
@@ -1,28 +1,38 @@
+import django
 import os
-from django.contrib.auth.models import User
-from player.models import Player
-from faction.models import Faction
-from setup.models import APIKey
-from awards.models import AwardsData
-from faction.models import FactionData
-from bazaar.models import BazaarData
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "yata.settings")
+django.setup()
```

Comment:
mmm I don't understand what you're doing here...

Reply:
Now that setup.py is loaded outside of the django env (manage.py) the django env still needs to be imported so that the imports/froms work correctly